themes/classic/templates/catalog/product.tpl

```diff
@@ -109,7 +109,7 @@
                         <p class="h4">{l s='This pack contains' d='Shop.Theme.Catalog'}</p>
                         {foreach from=$packItems item="product_pack"}
                           {block name='product_miniature'}
-                            {include file='catalog/_partials/miniatures/pack-product.tpl' product=$product_pack}
+                            {include file='catalog/_partials/miniatures/pack-product.tpl' product=$product_pack showPackProductsPrice=$product.show_price}
```

Comment:
You can avoid to use `showPackProductsPrice` and directly use `$product.show_price` in pack-product.tpl.
The reason is if someone use the `_partials/miniatures/pack-product.tpl` his template will raise an error because `$showPackProductsPrice` is undefined

Reply:
Well but if we change the $product var in this include it's a breaking change no ?

Reply:
Oh you're right, missreadding the `product=$product_pack` my bad :sweat_smile: